docs/index.rst

```diff
@@ -1370,13 +1391,381 @@ to setup alerts from there.
+
+   .. code-block:: text
+
+      kafka1            | 	confluent.telemetry.api.key = QX7X4VA4DFJTTOIA
+      kafka1            | 	confluent.telemetry.api.secret = [hidden]
+
+
+|crep|
+------
+
+#. If you have been running ``cp-demo`` for a long time, you may need to refresh your local token to log back into MDS:
+
+   .. sourcecode:: bash
+
+      ./scripts/helper/refresh_mds_login.sh
+
+#. Set the |crep| name.
+
+   .. code-block:: text
+
+      REPLICATOR_NAME=replicate-topic-to-ccloud
+
+#. Create a role binding to permit a new instance of |crep| to be submitted to the local connect cluster with id ``connect-cluster``.
+
+   Get the |ak| cluster ID:
+
+   .. literalinclude:: includes/get_kafka_cluster_id_from_host.sh
+
+   Create the role bindings:
+
+   .. code-block:: text
+
+      docker-compose exec tools bash -c "confluent iam rolebinding create \
+          --principal "User:connectorSubmitter" \
+          --role ResourceOwner \
+          --resource Connector:${REPLICATOR_NAME} \
+          --kafka-cluster-id ${KAFKA_CLUSTER_ID} \
+          --connect-cluster-id connect-cluster
+
+#. View the |crep| :devx-cp-demo:`configuration file|scripts/connectors/submit_replicator_to_ccloud_config.sh`. It is configured to copy from the |ak| topic ``wikipedia.parsed`` (on-prem) to the cloud topic ``wikipedia.parsed.ccloud.replica`` in |ccloud|. Note that it uses the local connect cluster (the origin site), so the |crep| configuration has overrides for the producer. The configuration parameters that use variables are read from the env variables that you sourced in an earlier step.
+
+#. Submit the |crep| connector to the local connect cluster.
+
+   .. code-block:: text
+
+      ./scripts/connectors/submit_replicator_to_ccloud_config.sh
+
+#. It takes about 1 minute to show up in the Connectors view in |c3|.  When it does, verify |crep| to |ccloud| has started properly, and there are now 4 connectors:
+
+   .. figure:: images/connectors-with-rep-to-ccloud.png
+
+#. Log into `Confluent Cloud <https://confluent.cloud>`__ UI and verify you see the topic ``wikipedia.parsed.ccloud.replica`` and its messages.
+
+#. View the schema for this topic that is already registered in |ccloud| |sr|. In ``cp-demo``, in the :devx-cp-demo:`Replicator configuration file|scripts/connectors/submit_replicator_to_ccloud_config.sh`, ``value.converter`` is configured to use ``io.confluent.connect.avro.AvroConverter``, therefore it automatically registers new schemas, as needed, while copying data. The schema ID in the on-prem |sr| will not match the schema ID in the |ccloud| |sr|. (See documentation for other :ref:`schema migration options <schemaregistry_migrate>`)
+
+   .. figure:: images/ccloud-schema.png
+
+Metrics
+-------
+
+.. include:: includes/metrics-api-intro.rst
+
+#. Get the current time minus 1 hour and plus 1 hour. These define a time interval when querying the Metrics API.
+
+   .. code-block:: bash
+
+      CURRENT_TIME_MINUS_1HR=$(date -Is -d '-1 hour')
```

Comment:
I get an error when trying to run this command on my mac. To remedy, I ran the command in the `tools` container. An approach of leveraging one of the running containers that you know has the right tools might be a good workaround for other mac users in my position. It might also be helpful here to say what time format you need so that people can create that if the command doesn't work. Just ideas.
```
localhost:cp-demo awalther$ CURRENT_TIME_MINUS_1HR=$(date -Is -d '-1 hour')
date: illegal option -- I
```

Reply:
@awalther28 thanks for surfacing this.  I'm working on a solution that uses the `tools` container (for now -- even though this container may be replaced in the future) for a consistent/reliable experience.

Reply:
@awalther28 latest commit addresses the timestamp issue.